pkg/apis/kubebind/v1alpha1/apiserviceexporttemplate_types.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2022 The Kube Bind Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// APIServiceExportTemplate specifies the resource to be exported.
+// It references the CRD to be exported along with additional resources that
+// are synchronized from and to the consumer cluster.
+//
+// +crd
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:scope=Namespaced,categories=kube-bindings
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Established",type="string",JSONPath=`.status.conditions[?(@.type=="Established")].status`,priority=5
+// +kubebuilder:printcolumn:name="Age",type="date",JSONPath=`.metadata.creationTimestamp`,priority=0
+type APIServiceExportTemplate struct {
```

Comment:
this is interesting, especially the selector. Don't think we have explored this side yet. Worth a discussion in the gdoc? My gut feeling is that this is really part of a demo backend, but could have different shape depending on context. For example the claims could differ depending on group membership or on input the UI.

Reply:
i am actually not understanding why this is needed.

Reply:
Suggestion: a different API group `example-backend.kube-bind.io/v1alpha1`